inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
-in your browser.
+* Download an example project for your first steps
+** Also on our website, clicking the tab „Example Projects“ on the very
+top leads you to the following page:
+https://inception-project.github.io/example-projects/ . To follow this
+Guide, please download the _Interactive Concept Linking_ project.
++
 
-== Your first annotation
+First Steps with INCEpTION
 
-After logging in, you will see the main menu. Click on *Projects* to go to the 
-*project management page* and there click on *create* to start a new project. Enter a name
-for your project, select the type *annotation project* and press *save*.
+* You may want to watch our tutorial video „Overview“ on how to create
+annotation:
+https://www.youtube.com/watch?v=wp4AN3p23mQ&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3&t=0s
+* In order to import our example project, after logging into INCEpTION,
+click on the blue folder on the top right (next to „Import project
+archives“) and browse for the example project you have downloaded
+before. Finally, click the grey **upload-**Butten right next to the blue
+folder to upload the selected project.
 
-Next, switch to the *Documents* tab and press *choose files* to select a plain text file from
-your local harddisk (the file should be in UTF-8 encoding). As part of the import process, 
-{product-name} automatically processes the file to identify sentence and token boundaries.
+* The project has now been added. Choose it by clicking on it and you
+will be on the dashboard of this project. Select *Annotation* on the top
+left and choose _pets1.tsv_ (the first file) in the window which now
+opened.
+* You will see several annotations already. If you import a documents
+without annotations yet, there will be none. But in the example
+projects, we have added some annotations already to show you some
+examples of annotations.
+* To create your first annotation, let‘s start with Part of Speech
+Tagging: Select *POS* from the *Layer* dropdown on the top right. When
+you click on a word in the text (e.g. „does“, in line 4), an annotation
+is immediately created and you can edit its details in the panels at the
+right (e.g. choose „VERB“ in the PosValue-dropdown to lable it as a
+verb).
 
-Use the *Home* link at the top of the screen to return to the main menu and select *Annotation*
-to open your text file in the annotation editor.
+Congratulations! You have created your first annotation!
 
-To create your first annotation, select *Named entity* from the *Layer* dropdown menu on the right
-side of the screen. Then, use the mouse to select a word in the *Annotation* area. When you release
-the mouse button, the annotation is immediately created and you can edit its details in the right
-sidebar.
+* Let‘s examine the right panel:
 
-Congratulations! You have created your first annotation project.
+* {blank}
+** You find the field „Layer“ on top and „Annotation“ below. In the
```

Comment:
Do you explain layers before?

Reply:
I'll put a layer explanation in a box right here.

Reply:
P.S.: the box comes not here but before.